runtime/description-generator.js

```diff
@@ -29,7 +30,7 @@ class DescriptionGenerator {
   getViewShortDescription(particleName, connectionName) {
     let description = this.getViewDescription(particleName, connectionName);
     if (description)
-      return description.replace(/\([a-zA-Z:, ]*\)/ig, '');
+      return description.replace(/ \([a-zA-Z0-9:, <>\/]*\)/ig, '');
```

Comment:
is this something we're going to do long-term? If so maybe a brief comment about what it does would be helpful?

Reply:
no, this is an ugly hack (but an old one, so i'll keep it for now). it will take some refactoring to get rid of, i'll do it later.

added comment and TODO for now.